src/main/java/io/strimzi/kafka/metrics/PrometheusMetricsReporterConfig.java

```diff
@@ -58,11 +54,7 @@ public class PrometheusMetricsReporterConfig extends AbstractConfig {
      * Default value for the allowlist configuration.
      */
     public static final String ALLOWLIST_CONFIG_DEFAULT = ".*";
-
-    /**
-     * Documentation for the allowlist configuration.
-     */
-    public static final String ALLOWLIST_CONFIG_DOC = "A comma separated list of regex Patterns to specify the metrics to collect.";
+    private static final String ALLOWLIST_CONFIG_DOC = "A comma separated list of regex Patterns to specify the metrics to collect.";
```

Comment:
I don't think Pattern deserves a capital.
```suggestion
    private static final String ALLOWLIST_CONFIG_DOC = "A comma separated list of regex patterns that specify the metrics to collect.";
```

Reply:
Capital P changed to lower case